packages/lib-classifier/src/store/Step.js

```diff
@@ -0,0 +1,9 @@
+import { types } from 'mobx-state-tree'
+
+const Step = types.model('Step', {
+  next: types.optional(types.string, ''), // We have this optionally to support recursive workflows
```

Comment:
I'm not sure. On Panoptes, I'm proposing a string of the step key be saved here, however, we can make it a reference here in the store? I'm having a bit difficulty about what exactly the model should do. Should it be a representation of what we expect from Panoptes or of what we plan on storing for the client app... I'm already planning on conversion of the `tasks` object to an ES6 Map, so it might be ok if we document that what is stored in the resource is just a string of the step key.